form/management/commands/queue_worker.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+import form.queue
+
+
+class Command(BaseCommand):
+    help = 'Runs a form data SQS queue worker'
+
+    @staticmethod
+    def setup_logging():
+        logger = logging.getLogger()
```

Comment:
not gonna work in mgm command - but I agree once again, see here: https://github.com/uktrade/directory-form-data/blob/add_async_db_save/form/queue.py#L14

This reconfigures logging so that all goes to stoudt, there's some funky stuff around that.